crypto.go

```diff
@@ -139,27 +155,58 @@ func verifyCertificateVerify(handshakeBodies []byte, hashAlgorithm HashAlgorithm
 	return errKeySignatureVerifyUnimplemented
 }
 
-func verifyClientCert(cert *x509.Certificate, roots *x509.CertPool) error {
+func loadCerts(rawCertificates [][]byte) ([]*x509.Certificate, error) {
+	certs := make([]*x509.Certificate, 0, 16)
```

Comment:
Seems like the size of this slice should be based off the length of rawCertificates

Reply:
Fixed!